### DISCUSSION TITLE
Update version available on apt?

### DISCUSSION THREAD

**Discussion by hdquemada** [neovim/neovim#25600] [General]
I downloaded neovim using apt, but it looks like a really old version. It's fine for my needs since I  only need to do basic text editing and am using vim/neovim as a LaTex editor too. However, is there any news about when there might be an upgrade? I suppose I could switch to a snap version if I felt a real need...

**Comment by clason** [comment]
No news. The Debian (volunteer!) maintainers are working on it as their free time permits.

**Comment by hdquemada** [comment]
Thanks! I appreciate their dedication. Wish I knew more about how to do
that kind of thing, or I'd help out.
---
Hector
